ariadne_codegen/client_generators/dependencies/async_base_client_open_telemetry.py

```diff
@@ -628,7 +635,11 @@ async def _send_subscribe_with_telemetry(
             )
 
     async def _handle_ws_message_with_telemetry(
-        self, root_span: Span, message: Data, websocket: WebSocketClientProtocol
+        self,
```

Comment:
Telemetry features are enabled only when `tracer` argument is not `None`. This client is also completely separate from the regular `AsyncBaseClient`.  Because of these 2 things there is some duplication of code and this change has to be placed also in the `_execute_ws` method.

Reply:
@mat-sop Added missing tests